examples/README.md

```diff
@@ -12,6 +12,13 @@ This document contains examples of various projects from Github.
 
 [![syncthing example](../images/syncthing.png)](https://raw.githubusercontent.com/TrueFurby/go-callvis/master/images/syncthing.png)
 
+```sh
+# Setup
```

Comment:
Perhaps it could be mentioned in this comment that this is special kind of setup because of syncthing build process.

Reply:
Done